static/src/stylesheets/amp/_d2-comments.scss

```diff
@@ -0,0 +1,50 @@
+/* Comments
+   ========================================================================== */
+
+.d2-comment-embedded {
+    background-color: #ffffff;
+    font-family: sans-serif;
+    overflow: hidden;
+    position: relative;
+    margin-bottom: 1.5em;
+    padding-top: 15px;
```

Comment:
Hi, thanks for pointing that out. I copied all this across from the settings used in the non-amp version of this css file. Not sure why the grid wasn't used originally. I can definitely update this file to use the grid settings now and can update the non-amp version of this file in a future pr.